src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -111,6 +111,10 @@ const OptionRowLHN = (props) => {
                             props.isFocused ? styles.sidebarLinkActive : null,
                             hovered && !props.isFocused ? props.hoverStyle : null,
                         ]}
+                        accessibilityLabel={optionItem.text}
+                        accessibilityRole="button"
```

Comment:
This is not in propTypes. should we add it?

Reply:
I think no because we just added specifying required values for PressableWithFeedback and defining that `OptionRowLHN` will always be a button and have the label `optionItem.text`